content/CORE/CORETutorials/Services/ConfiguringSMART.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Configuring S.M.A.R.T."
+weight: 90
+aliases: /core/services/smart/
+---
+
+{{< toc >}}
+
+[S.M.A.R.T.](https://en.wikipedia.org/wiki/S.M.A.R.T.) (Self-Monitoring, Analysis and Reporting Technology) is an industry standard for disk monitoring and testing.
+Disks can be monitored for problems using several different kinds of self-tests.
+
+Go to the **Services** page and find the **S.M.A.R.T.** entry.
+Click the <i class="material-icons" aria-hidden="true" title="Configure">edit</i> to configure the service.
+
+![ServicesSMARTOptions](/images/CORE/12.0/ServicesSMARTOptions.png "S.M.A.R.T. Options")
+
+Define a number of minutes for smartd to wake up and check for configured to run in **Check Interval** and select a **Power Mode** (tests only run with *Never*).
+
+Define the **Difference** in degrees of Celsius. S.M.A.R.T. will alert if the temperature of a drive changes by N degrees Celsius since the last report.
```

Comment:
```suggestion
Define the **Difference** in degrees of Celsius. S.M.A.R.T. sends alerts if the temperature of a drive changes by N degrees Celsius since the last report.
```